Formula/ruby@1.9.3.rb

```diff
@@ -0,0 +1,180 @@
+class RubyAT193 < Formula
```

Comment:
From a previous comment https://github.com/Homebrew/homebrew-core/pull/8767#issuecomment-272139269

Given you are the second person to find it misleading, and that ruby 1.9.0 is long gone. I guess we can just go with `ruby@1.8` and `class RubyAT19`.  It is less pure, but that should be good enough for nowadays usage.